pkg/console/operator/status.go

```diff
@@ -234,12 +235,8 @@ func (c *consoleOperator) ConditionsDefault(operatorConfig *operatorsv1.Console)
 		Reason:  reasonAsExpected,
 		Message: "As expected",
 	})
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:    operatorsv1.OperatorStatusTypeDegraded,
-		Status:  operatorsv1.ConditionFalse,
-		Reason:  reasonAsExpected,
-		Message: "As expected",
-	})
+
+	c.HandleDegraded(operatorConfig, "Default", nil)
```

Comment:
Looks promising 👍 We will need to revise the `consoleOperator`'s  `Condition*` methods...

Reply:
Yup. will continue on and hit all of these.  Is a bugzilla now, the condition statuses are not good enough.